### PR TITLE
[Platform.sh] Add Redis, mariadb 10.0 & readline php ext

### PR DESCRIPTION
**Pull request by andrerom**
- enable Redis
- Use MariaDB 10.0 instead of Mysql 5.5
- add readline php ext

ping @Crell *Note: Besides review and tests, could you check if port number in any cases is different than standard 6379? If so will need to add parameters for this.*

### CODE REVIEW

File: .platform/services.yaml [hunk 1]
```diff
@@ -1,3 +1,6 @@
 mysql:
```
Comment (Crell) [inline .platform/services.yaml]: You may also want to rename this service to mysqldb or similar, to avoid confusion.

**Comment by Crell** [comment]
You also need to add the rediscache relationship to the relationships block in .platform.app.yaml.

**Comment by andrerom** [comment]
> You also need to add the rediscache relationship to the relationships block in .platform.app.yaml.

What do you mean? It's added there.

**Comment by Crell** [comment]
You're correct.  I blanked on that hunk since it was so short.  False alarm!  (Comment about the port still stands.)

**Comment by andrerom** [comment]
@Crell fixed, ok?

**Comment by Crell** [comment]
Everything looks good to me visually.  If it compiles, :shipit:

**Comment by andrerom** [comment]
> Everything looks good to me visually. If it compiles, :shipit:

No idea, nothing to test against, thought you might have access to that. But thanks, merging.

**Pull request merged**